### PR TITLE
All Robot Framework Assignment

### PR DESCRIPTION
**Pull request by Anshulsatle**
Hello Team,

Please Review All Robot Framework Assignment

### CODE REVIEW

File: Anshul/Test/TC_MakeMyTrip.robot [hunk 1]
```diff
@@ -0,0 +1,12 @@
+*** Settings ***
+Documentation               This is Navigate
+
+
+
+*** Test Cases ***
+One way trip from Chennai to kolkata
+    Given Navigate to MakeMyTrip.com
+    When Select one way Trip  ${OneWay}
```
Comment (pcube0608) [inline Anshul/Test/TC_MakeMyTrip.robot]: Where have you defined this keyword?

`Select one way Trip`

File: Anshul/Resource/Amazon_Keyword.robot [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Goto all search categories
+     click element            ${Allcategories}
+     sleep                    10
+
+
+Toggle all categories one by one
+
+    @{ListOfCategories} =         get webelements           ${ListOfAllCategories}
+
+    FOR     ${Categories}       IN            @{ListOfCategorie}
```
Comment (pcube0608) [inline Anshul/Resource/Amazon_Keyword.robot]: Is this code working?
@{ListOfCategorie} is not defined. Its @{ListOfCategories}

File: Anshul/Test/TC_Yatra.robot [hunk 3]
```diff
@@ -0,0 +1,15 @@
+*** Settings ***
+Documentation    This Is a Yatra.com Site in which book round trip
+Library          SeleniumLibrary
+Resource         ../Resource/Yatra_Keyword.robot
+*** Test Cases ***
+Round Trip Flight Search
+    Given Navigate to Yatra.Com
+    And Accept the Cookeies
+    When Goto The Round Trip
+    Then Select Depature '${Depature.City}' to Arival City '${Arrival.City}'
```
Comment (pcube0608) [inline Anshul/Test/TC_Yatra.robot]: This keyword is not defined correctly. Is the code working?

`Select city '${Depature.City}' to '${Arrival.City}'`

**Review by pcube0608 — CHANGES_REQUESTED** [review]

**Review by Anshulsatle — COMMENTED** [review]
Check And Review!

**Review by pcube0608 — APPROVED** [review]
LGTM

**Review by ASDESHMUKH — APPROVED** [review]
LGTM!

**Pull request merged**